### PR TITLE
Use `SingleLineArrayWhitespace` to ensure single line arrays spacing

### PR DESCRIPTION
**Pull request by carusogabriel**
See https://github.com/slevomat/coding-standard/#slevomatcodingstandardarrayssinglelinearraywhitespace-.

### CODE REVIEW

File: tests/php-compatibility.patch [hunk 1]
```diff
@@ -28,7 +28,14 @@ index d1fdd9f..eab4288 100644
...
  tests/input/null_coalesce_operator.php                3       0
-@@ -33,15 +35,15 @@ tests/input/superfluous-naming.php                    11      0
+@@ -27,21 +29,22 @@ tests/input/optimized-functions.php                   1       0
+ tests/input/return_type_on_closures.php               21      0
+ tests/input/return_type_on_methods.php                17      0
+ tests/input/semicolon_spacing.php                     3       0
++tests/input/single-line-array-spacing.php             3       0
+ tests/input/spread-operator.php                       6       0
+ tests/input/static-closures.php                       1       0
+ tests/input/superfluous-naming.php                    11      0
```
Comment (malarzm) [inline tests/php-compatibility.patch]: what's happening in the diff here? Why so many additions?

File: tests/input/single-line-array-spacing.php [hunk 2]
```diff
@@ -0,0 +1,7 @@
+<?php
+
+declare(strict_types=1);
+
+$singleItem = [ $emptyArray ];
+
+$multiItems = [$emptyArray,$singleItem];
```
Comment (greg0ire) [inline tests/input/single-line-array-spacing.php]: Does it also fix a space before the comma?
```suggestion
$multiItems = [$emptyArray ,$singleItem];
```

File: tests/php-compatibility.patch (hunk 1 above)
Comment (malarzm) [reply tests/php-compatibility.patch]: Ah, same stuff as in the PR before, nvm

Comment (ostrolucky) [reply tests/php-compatibility.patch]: It's a .patch file, so that's what confuses github (and us). It's just a way patch files work.

**Comment by kukulich** [comment]
@carusogabriel I think you can enable “enableEmptyArrayCheck” as well

File: tests/input/single-line-array-spacing.php (hunk 2 above)
Comment (carusogabriel) [reply tests/input/single-line-array-spacing.php]: No, the fix if **after** the comma:
```diff
-$multiItems = [$emptyArray,$singleItem];
+$multiItems = [$emptyArray, $singleItem];
```

AFAIK we don't have a standard for spaces between variables and before commas

**Comment by carusogabriel** [comment]
@kukulich Thanks, but we already have that covered by `Squiz.Arrays.ArrayDeclaration.SpaceInEmptyArray` 😅

File: tests/input/single-line-array-spacing.php (hunk 2 above)
Comment (kukulich) [reply tests/input/single-line-array-spacing.php]: I think the sniff checks also spaces before comma.

Comment (carusogabriel) [reply tests/input/single-line-array-spacing.php]: Indeed:

```sh
 7 | ERROR | [x] Expected 0 spaces between "$emptyArray" and comma, 1 found.
```

Comment (greg0ire) [reply tests/input/single-line-array-spacing.php]: Let's add it to the test then?

**Comment by malarzm** [comment]
Thanks @carusogabriel!

**Pull request merged**